core/src/main/java/com/brandwatch/robots/net/FollowRedirectsFilter.java

```diff
@@ -110,6 +120,14 @@ private boolean isRedirectWithinLimits(ClientRequestContext requestContext, URI
         return true;
     }
 
+    private boolean isRedirectToExcludedDomain(String host) {
+        if (host == null) {
+            return false;
+        }
+        return excludedDomains.stream()
+            .anyMatch(excludedDomain -> host.equals(excludedDomain) || host.endsWith(excludedDomain));
```

Comment:
```suggestion
        if (host == null || excludedDomains.contains(host)) {
            return false;
        }
        return excludedDomains.stream()
            .anyMatch(host::endsWith);
```

Reply:
I'll adapt this into the implementation, thank you!